kolibri/tasks/management/commands/base.py

```diff
@@ -0,0 +1,27 @@
+from collections import namedtuple
+from django.core.management.base import BaseCommand
+
+
+Progress = namedtuple('Progress', ['progress', 'overall'])
```

Comment:
Could you outline the difference/usage of `progress` vs `overall` somewhere in here?